aqi_monitor/apps/monitor/tasks.py

```diff
@@ -1,10 +1,13 @@
 from __future__ import absolute_import, unicode_literals
 
-from celery.schedules import crontab
-from aqi_monitor.celery import app
 import requests
 import json
 
+from celery.schedules import crontab
+from aqi_monitor.celery import app
+from . import function_for_task as fun
```

Comment:
try to avoid using such aliases. it's applicable only when you're writing something temporary.

Reply:
Fixed